django/tests/otto/test_cleanup.py

```diff
@@ -249,8 +249,6 @@ def test_delete_old_chats_task(client, all_apps_user):
     chat = user.chat_set.first()
     chat_id = chat.id
     assert chat is not None
-    # Check that the chat.accessed_at is close to the start time
-    assert (chat.accessed_at - start_time).total_seconds() < 2
     time.sleep(3)
```

Comment:
Removed the part of this test that assumed it'd take less than 2 seconds to access a chat. It _usually_ would, but I figure it's neither consistent enough nor informative enough to be worth keeping.